packages/apps/webconnector/src/services/webConnector.ts

```diff
@@ -97,32 +103,32 @@ export default class webConnectorService extends TransactionBaseService {
 
       await this.oauthService_.update(oauth.id, { last_sync: new Date() });
 
-      const data = {
-        link: base_url,
-        link_id,
-        emit_event: false,
-        status: "success",
-      };
-      await this.organisationService.updateInstalledApp(
-        AppNameDefinitions.WEBCONNECTOR,
-        data
-      );
+      // const data = {
```

Comment:
remove this as well if its not in use

Reply:
This is database logic , work is in progress !!